test/fixer-test.js

```diff
@@ -3,7 +3,8 @@
 import assert from "assert";
 import preset from "../src/index";
 import { TextLintCore } from "textlint";
-import * as fs from "fs";
+import fs from "fs";
```

Comment:
https://github.com/Jam3/babel-plugin-static-fs/pull/17
This PR fix the workaround.